tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: Couchbase | Install Couchbase Repo
+  yum:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - http://packages.couchbase.com/releases/couchbase-release/couchbase-release-1.0-0-x86_64.rpm
```

Comment:
I'd argue this might want to be optional (for those who run their own mirrors).

Everyone seems to make this assumption and it's a PITA :)

Reply:
Ok cool makes sense will pass in as a var and default to this unless otherwise